printer_mods/arhodiewithsoul/Panel_Mounting/readme.md

```diff
@@ -0,0 +1,4 @@
+Contains new mounts for 3mm panels
+
+![panel_clip_corner_3mm_x12_001 (Large)](https://user-images.githubusercontent.com/80538348/116250886-1114a300-a766-11eb-8431-0a7ea3766e80.jpg)
+![panel_clip_corner_3mm_x12_002 (Large)](https://user-images.githubusercontent.com/80538348/116250889-1245d000-a766-11eb-9082-6018f1e6b8f3.jpg)
```

Comment:
Are the mid-panel clips different from stock?  If so, how are these better or different?